src/routes/spaces/address-books.controller.spec.ts

```diff
@@ -198,6 +202,24 @@ describe('AddressBooksController', () => {
     });
   });
 
+  describe('PUT /spaces/:spaceId/address-book', () => {
```

Comment:
These will be implemented on a forthcoming PR. However, the functionality is tested through `address-book-items.repository.integration.spec.ts`.